deno/private/toolchains_repo.bzl

```diff
@@ -89,7 +89,7 @@ toolchain(
     exec_compatible_with = {compatible_with},
     target_compatible_with = {compatible_with},
     toolchain = "@{user_repository_name}_{platform}//:deno_toolchain",
-    toolchain_type = "@contrib_rules_deno//deno:toolchain_type",
+    toolchain_type = "@aspect_rules_deno//deno:toolchain_type",
```

Comment:
Should this actually be using `{user_repository_name}` too (like the line above)? I'm pretty sure this currently makes the user's workspace name need to match whatever is hard coded here, meaning this is also a breaking change because it requires the user to rename the repository as `aspect_rules_deno`.